R/bundle.R

```diff
@@ -541,6 +541,15 @@ inferRPackageDependencies <- function(appMode, hasParameters, documentsHavePytho
   unique(deps)
 }
 
+# With shiny v1.7.2 or higher, renderPlot() defaults to ragg for png rendering
+# (if it's installed), so include it if it's installed locally (this way users
+# will get consistent PNG results without needing to add library(ragg) to the
+# app). https://github.com/rstudio/shiny/pull/3654
+shinyDeps <- function() {
+  include_ragg <- isAvailable("ragg") && isAvailable("shiny", "1.7.2")
```

Comment:
As written, it does not appear as if we can exercise this path, since the current Shiny on GitHub is identified as `1.7.1.9003`.

```r
rsconnect:::shinyDeps()
# => [1] "shiny"
```

Should we be checking that the Shiny version is greater than 1.7.1?

Reply:
Addressing this issue from here, as we are actively testing.

Reply:
@cpsievert - changed this to check for 1.7.1.9000 and higher.